src/jvm/test-integration/org/jetbrains/kotlinx/lincheck_test/guide/ConcurrentLinkedDequeTest.kt

```diff
@@ -39,5 +39,7 @@ class ConcurrentLinkedDequeTest {
 
     //@Test // TODO: Please, uncomment me and comment the line below to run the test and get the output
     @Test(expected = AssertionError::class)
-    fun modelCheckingTest() = ModelCheckingOptions().check(this::class)
+    fun modelCheckingTest() = ModelCheckingOptions()
+        .analyzeStdLib(true)
```

Comment:
There are also some stdlib tests in `org/jetbrains/kotlinx/lincheck_test/verifier/linearizability`, could you please also add this flag there. 

Also please re-check by project search if there are any other affected tests (search for `ConcurrentHashMap`, `ConcurrentLinkedDequeue`, etc).